### DISCUSSION TITLE
After connect crazyflie2.0 and crazyradio pa, report to many packets lost ?

### DISCUSSION THREAD

**Discussion by justinleeyang** [bitcraze/discussions#1584] [General]
Hi,

After connect crazyflie2.0 and crazyradio pa, report many packets lost ? 
When my hand is close to the red area in the picture below, the connection is normal without packet loss. 
In addition, if the Bluetooth connection is used with an Android phone, it is also normal.
![image](https://github.com/user-attachments/assets/927e736a-c0da-4b7f-9de1-1528d728f72c)

And I found similar problems in previous forums. How were they solved?
https://forum.bitcraze.io/viewtopic.php?t=297&hilit=To+many+packets+lost&start=20

**Comment by tobbeanton** [comment]
To me it sounds like the RFX2401C has not been switched on. Can you fill us in with more information? What HW are you using. What FW. Have you updated the FW and with what?

It could also be a "noisy" computer. Have you tired with another computer?

**Reply by justinleeyang** [reply]
My HW is crazyflie2.0 and crazyradio PA, FW is 2023.11 based Bitcraze VM, use the STLINK download the crazyflie2.0's firmware.

I try 2 computers, same problem.

**Reply by tobbeanton** [reply]
Can you please send some pitcures of the Crazyflie 2.0.
